sdk/storage/storage-file-datalake/src/DataLakeServiceClient.ts

```diff
@@ -48,6 +52,47 @@ export class DataLakeServiceClient extends StorageClient {
    */
   private blobServiceClient: BlobServiceClient;
 
+  /**
+   *
+   * Creates an instance of DataLakeServiceClient from connection string.
+   *
+   * @param {string} connectionString Account connection string or a SAS connection string of an Azure storage account.
+   *                                  [ Note - Account connection string can only be used in NODE.JS runtime. ]
+   *                                  Account connection string example -
+   *                                  `DefaultEndpointsProtocol=https;AccountName=myaccount;AccountKey=accountKey;EndpointSuffix=core.windows.net`
+   *                                  SAS connection string example -
+   *                                  `BlobEndpoint=https://myaccount.blob.core.windows.net/;QueueEndpoint=https://myaccount.queue.core.windows.net/;FileEndpoint=https://myaccount.file.core.windows.net/;TableEndpoint=https://myaccount.table.core.windows.net/;SharedAccessSignature=sasString`
```

Comment:
Check other datalake SDK align with datalake connection string format (for example, using "BlobEndpoint" as datalake endpoint)

Reply:
Oops, turns out .NET don't have this.